modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -400,6 +400,17 @@ definition {
 		}
 	}
 
+	macro addObjectValidationViaAPI {
+		JSONObject.addObjectValidation(
+			errorMessage = ${errorMessage},
+			labelName = ${labelName},
+			objectName = ${objectName},
+			script = ${script},
+			userEmailAddress = ${userEmailAddress},
+			userPassword = ${userPassword},
+			validationType = ${validationType});
+	}
+
```

Comment:
Hey @davisaints! I know we have been following this pattern already, but we would like to move away from creating a macro just to call another macro. Our tests can just directly call `JSONObject.addObjectValidation`. Let me know what you think. Thanks!

Reply:
Looks good, I didn't know we had stopped following this pattern.

Reply:
Looks good!